src/frontend/src/common/components/library/data_subview/index.tsx

```diff
@@ -259,7 +266,7 @@ const DataSubview: FunctionComponent<Props> = ({
           <Divider />
           <IconButton
             onClick={handleCreateTreeClickOpen}
-            disabled={isDownloadDisabled}
+            disabled={isCreateTreeDisabled}
```

Comment:
it seems like making this change causes `isDownloadDisabled` to not be read/used anywhere. is that intentional? If so, you may want to remove that state variable.